### PR TITLE
sensor report table

### PR DESCRIPTION
**Pull request by RadhikaPPatel**
## Purpose
Sensor report table

## Further info
Show tabular data for sensor report for a particular shipment.

### CODE REVIEW

File: src/midgard/pages/Shipment/ShipmentConstants.js [hunk 1]
```diff
@@ -238,6 +238,47 @@ export const custodyColumns = [
...
+    )
+  },
+  {
+    id: "humidity",
+    label: "Humidity",
+    minWidth: 150,
+  },
+  {
+    id: "temp",
+    label: "Temperature (\u00b0F)",
```
Comment (yasmin2496) [inline src/midgard/pages/Shipment/ShipmentConstants.js:277]: @RadhikaPPatel There will be additional columns coming here. Like, tilt, battery, pressure, shock. You will have to add those once BE is done with the changes

**Review by yasmin2496 — COMMENTED** [review]
Not merging it right now. As we have to do some more changes in columns.

File: src/midgard/pages/Shipment/ShipmentConstants.js (hunk 1 above)
Comment (RadhikaPPatel) [reply src/midgard/pages/Shipment/ShipmentConstants.js:277]: @yasmin2496 Yes. We can them the columns once the BE changes are done.

Comment (RadhikaPPatel) [reply src/midgard/pages/Shipment/ShipmentConstants.js:277]: Also to include these changes for new columns we will need to change a lot of places as sensor report response will be changed. So will put the new columns in while integrating for sensor report aggregation.

**Pull request merged**